### DISCUSSION TITLE
Play Noise

### DISCUSSION THREAD

**Discussion by Gloktober** [winstxnhdw/lc-hax#134] [General]
Hey winst !! Just to let you know that i think the play sound option doesn't work :(

**Comment by winstxnhdw** [comment]
Hey, it's intentional. It plays a virtual sound that only NPCs can hear. You should see Jeb or Eyeless Dogs react strongly to it.

**Reply by Gloktober** [reply]
oh awesome ! i thought it played a noise in their headset lol . So for the mask option , i need to have a mask in my hand ?

**Reply by winstxnhdw** [reply]
That's right
